src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -325,8 +325,12 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
     if (!this.userConfig.columns) {
       this.updateUserColumns();
     } else {
-      this.localColumns.forEach((c, i) => {
-        c.isHidden = this.userConfig.columns[i].isHidden;
+      this.userConfig.columns.forEach((col) => {
+        for (let i = 0; i < this.localColumns.length; i++) {
+          if (this.localColumns[i].prop === col.prop) {
+            this.localColumns[i].isHidden = col.isHidden;
+          }
```

Comment:
`i` is too generic for a variable name, so it would be good to have it defined like this or something else you prefer. WDYT?

```suggestion
        for (let index = 0; index < this.localColumns.length; index++) {
          if (this.localColumns[index].prop === col.prop) {
            this.localColumns[index].isHidden = col.isHidden;
          }
```

Reply:
> `i` is too generic for a variable name, so it would be good to have it defined like this or something else you prefer. WDYT?

Hi @nizamial09 

I've been thought to use shorter names for variables used in a local context where the usage and assignment are easily viewed. And have more descriptive names when you create fields or other variables where the context is larger.

I also saw multiple instances in this file where short one-character variables are used for local contexts.

But if the project requires longer names for this commit, I could change it.

Best regards
Daniel